MediaBrowser.Controller/Connect/IConnectManager.cs

```diff
@@ -7,34 +7,5 @@ namespace MediaBrowser.Controller.Connect
 {
     public interface IConnectManager
     {
-        /// <summary>
-        /// Gets the wan API address.
-        /// </summary>
-        /// <value>The wan API address.</value>
-        string WanApiAddress { get; }
```

Comment:
Getting WAN address can easily be done via DNS request to specific Google server if we want to keep the feature.

What is it used for anyway? Let someone know how to access the instance from outside? Looks like it should be a configuration option rather than autodetect...

Reply:
Yea, it shows on the admin dashboard. It's a "helpful" thing but TBH doesn't do much.

Reply:
Hmm, might as well choose to keep it for whatever reasons... but let user set it up.
For example, in my case I reverse-proxy it using nginx, so final WAN URL is impossible to construct from within the server.